_data/projects/MERN-Boilerplate.yml

```
@@ -0,0 +1,16 @@
+name: MERN-Boilerplate
+desc: This is boilerplate for MERN stack with integrations like Redux and SSR
+site: https://github.com/anikethsaha/MERN-Boilerplate
+tags:
+- Node.js
+- JavaScript
+- MongoDB
+- React
+- Express
+- Redux
+- Boilerplate
+- MERN
+- Server-side-rendering
+upforgrabs:
+  name: help wanted
+  link: https://github.com/anikethsaha/MERN-Boilerplate
```

This URL should point to a list of curated issues with the label `help wanted` but I don't see any issues created in the repository. Is this something you'd be able to do before merging? Otherwise it's not clear what new contributors could pick up...

---

Sure , I will mention you again after doing this.

---

@shiftkey  Please check and review now, I have added an issue with the same

---

Great! Now this link should go directly to that list:

```suggestion
  link: https://github.com/anikethsaha/MERN-Boilerplate/labels/help%20wanted
```